playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -663,6 +663,23 @@ EDXAPP_COMPREHENSIVE_THEME_SOURCE_REPO: ""
 # EDXAPP_COMPREHENSIVE_THEME_SOURCE_REPO
 EDXAPP_COMPREHENSIVE_THEME_VERSION: ""
 
+# supply a default site theme keyed to the main LMS domain
+# only works with EDXAPP_ENABLE_COMPREHENSIVE_THEMING set to true, 
+# otherwise, ignored - 
+# have to use this way of defining dict in string format
+# so the variable is interpolated in the key, but if not 
+# using variable key names a more standard format can be used
+EDXAPP_SITE_THEMES: "{
+    '{{ EDXAPP_LMS_SITE_NAME }}': {
+      'theme_name': '{{ EDXAPP_DEFAULT_SITE_THEME | default(\'edx-theme-codebase\', true) }}',
```

Comment:
@bryanlandia seems we have some syntax error here, during running the role i get: 
```
    '{{ EDXAPP_LMS_SITE_NAME }}': {
      'theme_name': '{{ EDXAPP_DEFAULT_SITE_THEME | default(\'edx-theme-codebase\', true) }}',
                                                             ^
We could be wrong, but this one looks like it might be an issue with
missing quotes.
```

Reply:
Thanks @amirtds - Hmm, may be an Ansible version issue then.  Can you tell me more about how you were running this, and versions?

Reply:
@bryanlandia Sure. 2 days ago i tried to make a push for Scamperand to PSU(they are eucalyptus) and i got the error i mentioned. i have venv for eucalyptus, Ansible version is `ansible 1.9.3-edx` and `ax, version 3.2.1`

Reply:
@amirtds - this PR is for Ficus.  I am not sure it works with Eucalyptus.  I just used it for a Ficus configuration without issue.  Could you try with Ficus?